public/docs/dart/latest/quickstart.jade

```diff
@@ -21,38 +21,40 @@ p.
     Create a new directory, and put a file named
     <code>pubspec.yaml</code> in it.
 
-  pre.prettyprint
-    code.
-      &gt; mkdir hello_world
-      &gt; cd hello_world
-      &gt; vim pubspec.yaml  # Use your favorite editor!
+  code-example(language="sh").
+    &gt; <span class="blk">mkdir hello_world</span>
+    &gt; <span class="blk">cd hello_world</span>
+    &gt; <span class="blk">vim pubspec.yaml</span>  # Use your favorite editor!
 
   p.
-    In <code>pubspec.yaml</code>, add the angular2 and browser packages as dependencies.
-    Angular 2 is changing rapidly, so specify an exact version:
-    <b>2.0.0-alpha.23</b>.
-
-  pre.prettyprint.linenums.lang-basic
-    code.
-      name: hello_world
-      version: 0.0.1
-      dependencies:
-        angular2: 2.0.0-alpha.23
-        browser: any
+    In <code>pubspec.yaml</code>,
+    specify the angular2 and browser packages as dependencies,
+    as well as the angular2 transformer.
+    Angular 2 is changing rapidly, so provide an exact version:
+    <b>2.0.0-alpha.25</b>.
+
+  code-example(language="yaml" format="linenums").
+    name: hello_world
+    version: 0.0.1
+    dependencies:
+      angular2: 2.0.0-alpha.25
+      browser: any
+    transformers:
+    - angular2:
+        entry_points: web/main.dart
   p.
     In the same directory, run <code>pub get</code>
     to install the angular2 and browser packages
     (along with the packages they depend on):
 
-  pre.prettyprint.lang-basic
-    code.
-      &gt; pub get
+  code-example(language="sh").
+    &gt; <span class="blk">pub get</span>
 
-  // PENDING: Create template? Link to pub/pubspec docs?
-  // Is browser really needed?
+  //- PENDING: browser: any -> ???
```

Comment:
I'm checking with Natalie about what we should use instead of "browser: any", since "any" is now discouraged.